lte/gateway/fabfile.py

```diff
@@ -535,6 +541,13 @@ def _dist_upgrade():
     run('sudo apt-get update')
     run('sudo DEBIAN_FRONTEND=noninteractive apt-get -y dist-upgrade')
 
+def _from_registry():
```

Comment:
shouldn't you be passing the hash here?

Reply:
so right now, I'm dynamically fetching it, but yeah it's WIP